pkg/cli/cmd/radinit/aws.go

```diff
@@ -123,3 +169,19 @@ func (r *Runner) buildAWSRegionsList(listRegionsOutput *ec2.DescribeRegionsOutpu
 
 	return regions
 }
+
+func (r *Runner) selectAwsCredentialKind() (string, error) {
+	credentialKinds, err := r.buildAwsCredentialKind()
+	if err != nil {
+		return "", err
+	}
+
+	return r.Prompter.GetListInput(credentialKinds, selectAwsCredentialKindPrompt)
+}
+
+func (r *Runner) buildAwsCredentialKind() ([]string, error) {
```

Comment:
This returns a list of available AWS credential kinds. Should we change the name? Also, can this ever return an error?

Reply:
updated it to remove error from the return type. 
and for the name today we follow the similar format in azure provider:
https://github.com/radius-project/radius/blob/b7484430568763f05fba769b325e6f0471092fcb/pkg/cli/cmd/radinit/azure.go#L307